src/content/SectionPopup/SectionInfoBody.tsx

```diff
@@ -41,14 +20,8 @@ const SectionInfoBody = ({ selectedSection }: SectionInfoProps) => {
       )}
       <hr />
       <div className="SectionPopupDetails">{selectedSection?.name}</div>
-      {isLoading || supplementaryData === null ? (
-        <div>Loading Data...</div>
-      ) : (
-        <>
-          <InstructorComponent instructors={supplementaryData.instructors} />
-          <LocationComponent locations={supplementaryData.locations} />
-        </>
-      )}
+      <InstructorComponent selectedSection={selectedSection} />
+      <LocationComponent selectedSection={selectedSection} />
```

Comment:
Why not just pass in `selectedSection.instructors` and `selectedSection.locations`? These child components don't need access to any other attributes.